config/manager/manager.yaml

```diff
@@ -23,12 +23,22 @@ spec:
         control-plane: controller-manager
     spec:
       securityContext:
-        runAsNonRoot: true
+        runAsNonRoot: false
+      volumes:
+      - name: ssh
+        secret:
+          defaultMode: 384
+          optional: true
+          secretName: ssh-key-secret
```

Comment:
peerpod-ctrl still doesn't work with ssh authentication right? i think you should have comment on this and point to https://github.com/confidential-containers/cloud-api-adaptor/issues/1241